### PR TITLE
Problem: the new merged gorc is not tested

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update gravity-bridge dependency

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/test_gravity.py [hunk 1]
```diff
@@ -39,11 +38,7 @@ def cronos_crc20_abi():
 
 def gorc_config(keystore, gravity_contract, eth_rpc, cosmos_grpc, metrics_listen):
     return {
-        "keystore": InlineTable(
-            {
-                "File": str(keystore),
-            }
-        ),
+        "keystore": str(keystore),
```
Comment (yihuang) [inline integration_tests/test_gravity.py:41]: will need to change back after aws secret manage feature is merged.

File: nix/sources.json [hunk 2]
```diff
@@ -37,15 +37,15 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "gravity-bridge": {
-        "branch": "cronos",
+        "branch": "thomas/add-ethermint-support",
```
Comment (yihuang) [inline nix/sources.json:40]: TODO, update after merged.

File: integration_tests/contracts/contracts/Gravity.sol [hunk 3]
```diff
@@ -1,13 +1,27 @@
-pragma solidity ^0.6.6;
```
Comment (yihuang) [inline integration_tests/contracts/contracts/Gravity.sol:1]: I copied gravity contracts from: https://github.com/PeggyJV/gravity-bridge/tree/main/solidity/contracts

File: integration_tests/test_gravity.py (hunk 1 above)
Comment (JayT106) [reply integration_tests/test_gravity.py:41]: which repo will integrate the `aws secret manager` feature? Is the `gravity-bridge`?

**Comment by JayT106** [comment]
Do all files in`integration_tests/contracts` folder merge from the upstream repo?

**Comment by yihuang** [comment]
> Do all files in`integration_tests/contracts` folder merge from the upstream repo?

Only the `Gravity.sol` and `CosmosToken.sol`, the other contracts are created for testing purposes.

**Review by JayT106 — APPROVED** [review]
LGTM

**Pull request merged**